src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -121,9 +121,13 @@ void EditShortcutModel::checkNewSequenceForConflicts()
     for (const QVariant& shortcut : m_potentialConflictShortcuts) {
         QVariantMap map = shortcut.toMap();
 
-        if (map.value("sequence").toString() == input) {
-            m_conflictShortcut = map;
-            return;
+        std::vector<std::string> toCheckSequences = Shortcut::sequencesFromString(map.value("sequence").toString().toStdString());
+
+        for (const std::string& toCheckSequence : toCheckSequences) {
+            if (input.toStdString() == toCheckSequence) {
```

Comment:
Please convert the "input" string outside the loops

Reply:
Done, also made it const